clients/gateway/errors.go

```diff
@@ -0,0 +1,12 @@
+package gateway
+
+var (
+	ErrTransactionLimitExceeded        = "StarknetErrorCode.TRANSACTION_LIMIT_EXCEEDED"
+	ErrOutOfRangeFee                   = "StarknetErrorCode.OUT_OF_RANGE_FEE"
+	ErrInvalidTransactionVersion       = "StarknetErrorCode.INVALID_TRANSACTION_VERSION"
+	ErrInvalidProgram                  = "StarknetErrorCode.INVALID_PROGRAM"
+	ErrDeprecatedTransaction           = "StarknetErrorCode.DEPRECATED_TRANSACTION"
+	ErrInvalidCompiledClassHash        = "StarknetErrorCode.INVALID_COMPILED_CLASS_HASH"
+	ErrCompilationFailed               = "StarknetErrorCode.COMPILATION_FAILED"
+	ErrUnauthorizedEntryPointForInvoke = "StarknetErrorCode.UNAUTHORIZED_ENTRY_POINT_FOR_INVOKE"
+)
```

Comment:
This should be `const`. Also, where will these errors be used?

Reply:
Currently they are not needed, so I've removed them. We might need them in the future if we implement a gateway test Server